go.mod

```diff
@@ -9,5 +9,6 @@ require (
 	github.com/go-toolsmith/astp v0.0.0-20180903215135-0af7e3c24f30
 	github.com/go-toolsmith/strparse v0.0.0-20180903215201-830b6daa1241
 	github.com/go-toolsmith/typep v0.0.0-20181030061450-d63dc7650676
+	github.com/mattn/goveralls v0.0.2 // indirect
```

Comment:
I'm not a big fun of an another dependency (which is a tool for testing only).
But for now it's ok.